packages/validator/src/services/validatorStore.ts

```diff
@@ -290,6 +299,27 @@ export class ValidatorStore {
     };
   }
 
+  async signValidatorRegistration(
+    pubkey: BLSPubkey,
+    feeRecipient: ExecutionAddress,
+    gasLimit: number,
+    _slot: Slot
+  ): Promise<bellatrix.SignedValidatorRegistrationV1> {
+    const timestamp = Math.floor(Date.now() / 1000);
+    const validatorRegistation: bellatrix.ValidatorRegistrationV1 = {
+      feeRecipient,
+      gasLimit,
+      timestamp,
+      pubkey,
+    };
+    const domain = computeDomain(DOMAIN_APPLICATION_BUILDER, this.config.GENESIS_FORK_VERSION, ZERO_HASH);
```

Comment:
Why do we go with genesis fork here?

shouldn't it be `const domain = this.config.getDomain(DOMAIN_APPLICATION_BUILDER, slot);`

Reply:
thats as per the spec, i think the idea being that the validator registration doesn't need any fork safeguards